pkg/asset/internal/templates.go

```diff
@@ -1079,13 +1079,38 @@ spec:
         - name: flannel-cfg
           mountPath: /etc/kube-flannel/
       - name: install-cni
-        image: {{ .Images.Busybox }}
-        command: [ "/bin/sh", "-c", "set -e -x; TMP=/etc/cni/net.d/.tmp-flannel-cfg; cp /etc/kube-flannel/cni-conf.json ${TMP}; mv ${TMP} /etc/cni/net.d/10-flannel.conf; while :; do sleep 3600; done" ]
+        image: {{ .Images.Alpine }}
+        command: 
```

Comment:
@abhinavdahiya Created a repo for the image here: https://github.com/coreos/flannel-cni 

/cc @aaronlevy @rimusz